libraries/botbuilder-core/src/testAdapter.ts

```diff
@@ -310,6 +310,33 @@ export class TestAdapter extends BotAdapter implements ExtendedUserTokenProvider
         }
     }
 
+    public async getTokenStatus(context: TurnContext, userId: string, includeFilter?: string, oAuthAppCredentials?: any): Promise<any[]> {
+        const filter = (includeFilter ? includeFilter.split(',') : undefined);
+        const match = this._userTokens.filter(x => x.ChannelId === context.activity.channelId 
+                                            && x.UserId === context.activity.from.id 
```

Comment:
If context.activity or context.activity.from is falsy this method will throw an error, can you add falsy checks when the method is invoked? (And tests too 😄)

Reply:
Sure, but this is test code ... not the product.  It feels weird to writes tests validating parameters of test code.